src/routes/safe/components/Transactions/TxsTable/columns.tsx

```diff
@@ -56,8 +68,19 @@ export const getTxAmount = (tx, formatted = true) => {
   return getAmountWithSymbol({ decimals, symbol, value }, formatted)
 }
 
-const getIncomingTxTableData = (tx) => ({
-  [TX_TABLE_ID]: tx.blockNumber,
+interface TableData {
+  amount: string
+  cancelTx?: Transaction
+  date: string
+  dateOrder?: number
+  id: string
+  status: string
+  tx?: Transaction
+  type: any
+}
+
+const getIncomingTxTableData = (tx: Transaction): TableData => ({
+  [TX_TABLE_ID]: tx.blockNumber?.toString() ?? '',
```

Comment:
what does happen to the table if there are multiple transactions with ids equal to ''?

Reply:
just :boom:

In fact, will happen what happens to rows with the same id. If you expand one, all the others will expand as well.

I thought about that when working on this and thought that the `blockNumber` isn't a proper id either, as it can be duplicated too.

Reply:
Ok, this is odd... it seems that the `id` is replaced by the function [`getTxTableData`](https://github.com/gnosis/safe-react/blob/b4ce1ff679fc31c2607a074f28bf73bba777ad97/src/routes/safe/components/Transactions/TxsTable/index.tsx#L38-L63)

Actually, so it won't affect the transactions table the value set in there.